docs/howtos/layers/labels.md

```diff
@@ -76,7 +76,7 @@ layer:
 
   Use this tool to manually erase a label on the `labels layer`. Other layers
   will not be affected. The label eraser tool looks like this:
-  ![image: eraser tool](../../images/labels-layer-eraser.png)
+  ![image: eraser tool](../../images/layers-labels-eraser.png)
```

Comment:
This doesn't appear correct, see:
[https://github.com/napari/docs/blob/main/docs/images/labels-layer-eraser.png](https://github.com/napari/docs/blob/main/docs/images/labels-layer-eraser.png)

It's causing a warning:
https://github.com/napari/docs/actions/runs/10563443854/job/29263666686?pr=484#step:9:1436
Let's revert?
```suggestion
  ![image: eraser tool](../../images/labels-layer-eraser.png)
```

Reply:
Oh man, search and replace all strikes again 🙈 Will fix